rpcserver.go

```diff
@@ -2049,9 +2049,31 @@ func (r *rpcServer) FundVirtualPsbt(ctx context.Context,
 
 	case req.GetRaw() != nil:
 		raw := req.GetRaw()
+		prevIDs := []asset.PrevID{}
 		if len(raw.Inputs) > 0 {
-			return nil, fmt.Errorf("template inputs not yet " +
-				"supported")
+			for _, input := range raw.Inputs {
+				// Create a new chainhash.Hash and set its bytes
+				var hash chainhash.Hash
+				err := hash.SetBytes(input.Outpoint.Txid)
+				if err != nil {
+					return nil, fmt.Errorf(
+						"invalid Txid length: %w", err,
+					)
+				}
+				// Decode the input into an asset.PrevID
```

Comment:
nit: if a comment isn't at the beginning of a new block/indentation, we usually add an empty line before to give some breathing space.